src/ipc/eve_types.rs

```diff
@@ -1205,7 +1203,7 @@ pub struct VmConfig {
     pub extra_args: String,
     pub boot_loader: String,
     #[serde(rename = "CPUs")]
-    pub cpus: String,
+    pub cpus: Option<String>,
```

Comment:
Is this similar to omitempty in golang? Like if it's missing I'll just have None and if I didn't have it it'd default value it (or raised the error?)

Reply:
@uncleDecart yes, the filed can be 'null' in json and Option converts it to None, otherwise you'll get 'string expected but null found'

Reply:
Okay got it, thanks!